WordPress/Classes/Models/Post.m

```diff
@@ -21,6 +22,79 @@ - (void)didTurnIntoFault
     self.specialType = nil;
 }
 
+#pragma mark - Super methods
+
+- (NSArray *)availableStatuses
+{
+    NSArray *defaultStatuses = [super availableStatuses];
```

Comment:
Given the current logic of this method, this line doesn't really need to run in all cases.  For instance: this variable isn't used at all inside the IF block below - so it could be moved below the conditional block.